src/supermarket/db/migrate/20161019183011_create_publish_quality_metric.rb

```diff
@@ -0,0 +1,5 @@
+class CreatePublishQualityMetric < ActiveRecord::Migration
+  def change
+    publish_qm = QualityMetric.create!(name: 'Publish', admin_only: true)
```

Comment:
Maybe the difference between this QualityMetric and the ones created previously is that this is a type of metric that didn't exist before whereas the others existed previously as columnar data that needed to be transformed into instances of a thing that needed to exist at the time of transformation.